tests/testWPAuth0DbMigrations.php

```diff
@@ -121,7 +121,7 @@ public function testThatV21UpdatesCdnUrl() {
 	 */
 	public function testThatV21UpdatesWle() {
 		$test_version = 21;
-		$db_manager = new WP_Auth0_DBManager( self::$opts );
+		$db_manager   = new WP_Auth0_DBManager( self::$opts );
```

Comment:
Intentional?

Reply:
Yes, aligns the equals sign (auto format)